src/rgw/rgw_sal.h

```diff
@@ -1530,6 +1530,8 @@ class DriverManager {
   static rgw::sal::Driver* get_storage(const DoutPrefixProvider* dpp,
 				      CephContext* cct,
 				      const Config& cfg,
+				      boost::asio::io_context& io_context,
+				      rgw::SiteConfig& site_config,
```

Comment:
as discussed in slack, can this be a const-ref? drivers shouldn't be able to mutate this

Reply:
Done